plugins/async-node/core/src/index.ts

```diff
@@ -44,7 +44,7 @@ export class AsyncNodePlugin implements PlayerPlugin {
   }
 
   public readonly hooks = {
-    onAsyncNode: new AsyncParallelBailHook<[Node.Async], any>(),
```

Comment:
Hmm - the incoming change from `main` should be accepted here, as a part of:
#371